src/game/gui/textslider.c

```diff
@@ -16,35 +16,37 @@ typedef struct {
     int *pos;
     int has_off;
     int positions;
+    char txt[20];
 
     void *userdata;
     textslider_slide_cb slide;
 } textslider;
 
-static void textslider_render(component *c) {
+static void render_text(component *c) {
```

Comment:
Please rename this method to something that does not contain the word "render."

textslider_format? textslider_updatestr?